packages/playht/src/grpc-client/tts-stream-source.ts

```diff
@@ -105,3 +147,20 @@ export class TTSStreamSource implements UnderlyingByteSource {
     return false;
   }
 }
+
+/**
+ * Enumerates a streaming congestion control algorithms, used to optimize the rate at which text is sent to PlayHT.
+ */
+export enum CongestionCtrl {
```

Comment:
I had to copy and paste this here to get around dependency cycles surfaced by yarn verify.